File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domene/PubliseringsdatoerDatauthentingFeil.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domene;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

@ResponseStatus(code = HttpStatus.INTERNAL_SERVER_ERROR)
public class PubliseringsdatoerDatauthentingFeil extends RuntimeException {

  public PubliseringsdatoerDatauthentingFeil(String message) {
    super(message);
  }
}

File: src/main/kotlin/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/publiseringsdatoApi/PubliseringsdatoerDatauthentingFeil.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.publiseringsdatoApi

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus

@ResponseStatus(code = HttpStatus.INTERNAL_SERVER_ERROR)
class PubliseringsdatoerDatauthentingFeil(message: String?) : RuntimeException(message)
